onadata/apps/logger/data_migration/migrate_data.py

```diff
@@ -49,11 +49,13 @@ def migrate(self):
 class SurveyFieldsHandler(object):
     """Handle fields operations."""
     def __init__(self, migration_decisioner):
+        # FIXME: perhaps survey_tree should also be a class attribute
```

Comment:
what about this one?

Reply:
After reconsidering it, it turns out it is not that obvious to make it a class attrib. Removing